salt/modules/mac_group.py

```diff
@@ -55,6 +66,8 @@ def add(name, gid=None, **kwargs):
         )
     if gid is not None and not isinstance(gid, int):
         raise SaltInvocationError("gid must be an integer")
+    if "non_unique" in kwargs:
```

Comment:
Why are we adding this for the mac module? I'm not seeing why this is necessary?

Reply:
Because `non_unique` only works for Linux via the underlying `groupadd` module at this time. If someone tries to use that parameter for `group.present` on Mac, it would silently ignore the option without this message... which is not ideal.